service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java

```diff
@@ -128,10 +128,18 @@ private List<SiteEntry> getSiteEntries(int siteId) {
   @Override
   public GetSiteResponse getSite(int siteId) {
     SitesRecord sitesRecord = db.selectFrom(SITES).where(SITES.ID.eq(siteId)).fetchOne();
+    SiteEntriesRecord siteEntriesRecord =
```

Comment:
🔧 Move this query below the check on 137. The if(site_XXX == null) is essentially a post-condition check for the query on 130. You break the cohesion between these two lines by shoving this query between them.

🏕️ Honestly, both of these should be put into their own methods.